core/commands/refs.go

```diff
@@ -274,43 +286,53 @@ func (rw *RefWriter) writeRefsRecursive(n ipld.Node) (int, error) {
 	return count, nil
 }
 
-func (rw *RefWriter) writeRefsSingle(n ipld.Node) (int, error) {
-	c := n.Cid()
-
-	if rw.skip(c) {
-		return 0, nil
+// visit returns two values:
+// - first indicates if we should keep traversing the DAG.
+// - second indicates if the given Cid should be printed to the user.
```

Comment:
"indicates" is ambiguous. In the first case, it means "is true" and, in the second case, "is false".

Personally, I'd rather:

1. Say "is true" (explicitly).
2. Invert the second case (i.e., true means print).